hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/RollbackToInstantTimeProcedure.scala

```diff
@@ -38,7 +37,8 @@ class RollbackToInstantTimeProcedure extends BaseProcedure with ProcedureBuilder
     ProcedureParameter.required(1, "instant_time", DataTypes.StringType, None))
 
   private val OUTPUT_TYPE = new StructType(Array[StructField](
-    StructField("rollback_result", DataTypes.BooleanType, nullable = true, Metadata.empty))
+    StructField("rollback_result", DataTypes.BooleanType, nullable = true, Metadata.empty),
+    StructField("instant_time", DataTypes.StringType, nullable = true, Metadata.empty))
```

Comment:
> Like `rollback_to_savepoint`, the request argument of `rollback_to_instant` has the `instant_time`, therefore the return result doesn't need the `instant_time`.

This will iteratively output multiple lines of rollback instants.